common/app/templates/inlineJS/nonBlocking/detectAdblock.scala.js

```diff
@@ -0,0 +1,24 @@
+try {
+    ((document, window) => {
+        var ad = document.createElement('div');
+        ad.style.position = 'absolute';
+        ad.style.left = '-9999px';
+        ad.style.height = '10px';
+        ad.innerHTML = '&nbsp;';
+        ad.setAttribute('class', 'ad_unit');
+
+        window.requestAnimationFrame(() => {
+            document.body.appendChild(ad);
+
+            window.requestAnimationFrame(() => {
```

Comment:
after a few good chats offline, I'm going to update this account fully for the async nature of the test (reading and writing to the DOM), but I won't be changing the technique. 

that's a whole other can of worms, and moving this is really only to allow us to consolidate the omniture calls, rather than improve detection. 

a good discussion and v interesting area though for sure...